### PR TITLE
Fixed updates from update_timer preventing nodes from being renamed

### PR DESCRIPTION
**Pull request by Piet-G**

### CODE REVIEW

File: editor/scene_tree_editor.cpp [hunk 1]
```diff
@@ -483,6 +483,15 @@ void SceneTreeEditor::_test_update_tree() {
 	tree_dirty = true;
 }
 
+void SceneTreeEditor::_attempt_configuration_update() {
+	if (tree->is_editing()) {
+		update_timer->start();
+	}
+	else{
```
Comment (akien-mga) [inline editor/scene_tree_editor.cpp]: Style issue, it should be `} else {` all on the same line.

Comment (DualMatrix) [reply editor/scene_tree_editor.cpp]: Fixed

**Comment by akien-mga** [comment]
What bug does this aim to fix? I've tried to rename various nodes in the scene tree, including batch rename, and did not experience any issue.

**Comment by akien-mga** [comment]
Moving to 3.2 milestone as we're about to enter the beta stage and *release freeze* for Godot 3.1. Only bug fixes against issues of the 3.1 milestone (or enhancements to core features of the 3.1 roadmap) will be considered until 3.1-stable is released.

Note: If it's merged after 3.1-stable is released, this change may be considered for cherry-picking into a later 3.1.x release.

**Comment by DualMatrix** [comment]
I'm not exactly sure what you need to do to reproduce this exactly. When I used a couple of nodes with  custom configuration warnings in a scene it would prevent nodes from being renamed.

I would say this is an issue that really needs to be fixed before 3.1 because it makes working with nodes really unusable.

**Comment by DualMatrix** [comment]
So I tried reproducing this but I couldn't really get it to fail again. I've since worked on the project I got this issue in so I'm not exactly sure anymore on what exactly made it fail. 

I still think there's something wrong with the trees being updated while editing but without being able to reproduce it that doesn't mean much :/ 

I'll report back as soon as I encounter it again.

**Comment by mhilbrunner** [comment]
So could you reproduce it? If not, maybe we should close it and archive it, but keep the branch around in case it resurfaces :)

**Comment by akien-mga** [comment]
Closing for now as per above comments.

**Pull request closed without merging**